database/seedGenerator.js

```diff
@@ -0,0 +1,135 @@
+const faker = require('faker');
+// const fs = require('fs');
+const model = require('./db.js');
+
+const USERS_HEADERS = "userId,userName";
+const PROJECTS_HEADERS = "projectId,projectName,ownerId";
+const UPDATES_HEADERS = "id,title,postedBy,projectId,body,likes,pubDate,createdAt,updatedAt";
+const LIKES_LIMIT = 500;
+const BODY_PARAGRAPH_MAX = 15;
+const BODY_PARAGRAPH_MIN = 2;
+const MIN_UPDATES = 0;
+const MAX_UPDATES = 10;
+const KICKSTARTER_FOUNDED = new Date(2009, 3, 28);
+
+let updateCount = 0;
+
+function getName() {
+  return faker.name.findName();
+}
+
+function getLikes() {
+  return Math.floor(Math.random() * LIKES_LIMIT);
+}
+
+function getUpdateBody() {
+  // from 2 to 15 paragraphs
+  const paragraphCount =
+    Math.floor(Math.random() * BODY_PARAGRAPH_MAX - BODY_PARAGRAPH_MIN) + BODY_PARAGRAPH_MIN;
+  return faker.lorem.paragraphs(paragraphCount);
+}
+
+function getTitle() {
+  return faker.hacker.phrase();
+}
+
+function randomNum(min, max) {
+  return Math.floor(Math.random() * (max + 1 - min)) + min;
+}
+
+function randomDate(start, end) {
+  return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+}
+
+function intPadLeft(num, base, chr) {
+  const len = String(base || 10).length - String(num).length + 1;
```

Comment:
I got a little confused trying to follow what this helper function was doing. Is it just so you could get the correct date format for SQL? (which does not look fun by the way...)

Reply:
Yes, that's what it's doing. I'll refactor to make it more clear.